slam_toolbox:
  ros__parameters:
    use_sim_time: true

    # Frames: match your project
    odom_frame: odom
    map_frame: map
    base_frame: base_link

    # Topic: match your bridge + sensor
    scan_topic: /scan

    # Make message filter more tolerant in sim
    transform_timeout: 0.5
    tf_buffer_duration: 30.0

    # Mapping behavior (safe defaults)
    mode: mapping
    resolution: 0.05
    max_laser_range: 12.0
    minimum_time_interval: 0.0
